Week2/task1.js

```diff
@@ -0,0 +1,7 @@
+console.log(`More JavaScript`);
+console.log(`
+Task 1`); // a function which adds three given arguments.
+function sum(a, b, c) {
+  return (sum = a + b + c);

```

Comment:
You don't need to call the function in here, `return a + b + c` will work just fine.

Reply:
Yes, I did it now.
Thanks.